playbooks/setup_jenkins_slave.yml

```diff
@@ -2,6 +2,17 @@
 - hosts: job_nodes
   user: root
   vars:
+    ansible_ssh_extra_args: >
+      -o UserKnownHostsFile=/dev/null
+      -o StrictHostKeyChecking=no
+      -o ServerAliveInterval=64
+      -o ServerAliveCountMax=1024
+      -o Compression=no
+      -o TCPKeepAlive=yes
+      -o VerifyHostKeyDNS=no
+      -o ForwardX11=no
+      -o ForwardAgent=yes
+      -T
```

Comment:
@hughsaunders Do you think that these parameters should be added to group_vars to make them used globally by all rpc-gating playbooks or do you think they're better left in this playbook exclusively?

Reply:
I personally think this should be done for all RPC playbooks however to limit impact I've kept it in the one playbook. @odyssey4me @hughsaunders if you'd like to have this in the group vars I'd be happy to move it there.

Reply:
I'm happy for that to go into group vars.